### ISSUE TITLE
On Windows : asyncio\events.py", line 481, in add_signal_handler raise NotImplementedError

### ISSUE THREAD

**Issue by pulse-mind** [XKNX/xknx#133]
Hello,

When I run an example (example_telegram_monitor.py) under windows I get this error. 

Thank you
Fred

**Comment by pulse-mind** [comment]
Thank you for your answer. 
How I do that ? "did you try to connect directly via a configured tunnel instead of using the gateway scanner"

**Comment by Julius2342** [comment]
try to pass a connection config where you have set the mode to "Tunnel" and have speficied a fixed IP for the gateway.

https://github.com/XKNX/xknx/blob/master/xknx/xknx.py#L63

**Comment by pulse-mind** [comment]
Thank you. I was already using the ConnectionConfig for setting IP and Port. I changed AUTOMATIC to TUNNELING. The error is still the here
```
asyncio\events.py", line 481, in add_signal_handler
    raise NotImplementedError
NotImplementedError
Task was destroyed but it is pending!
```

**Comment by Julius2342** [comment]
uh .. what python version do you use?

The error is when registering the sig-int handler (which helps cleaning up after having received a control-C signal). May I ask you to checkout xknx, use it with PYTHON_PATH and comment out this line?

**Comment by pulse-mind** [comment]
I am using Python 3.6

**Comment by Julius2342** [comment]
https://github.com/crossbario/autobahn-python/commit/231c204eda00b139d259073bcfe864bf8393fc2f

looks like windows does not support signals ...

May you try not using daemon_mode (`daemon_mode=False`) and instead simply add an endless loop with an async-sleep?

**Comment by Julius2342** [comment]
We also could do something like:

```
try:
    loop.add_signal_handler(signal.SIGTERM, loop.stop)
except NotImplementedError:
    pass  # Ignore if not implemented. Means this program is running in windows.
```

**Comment by pulse-mind** [comment]
This is what I done locally

```
from sys import platform
```

```
if platform == "win32":
            self.logger.warning('Windows does not support signals')
        else:
            self.loop.add_signal_handler(signal.SIGINT, sigint_handler)
```

Then it works fine

**Comment by pulse-mind** [comment]
Do you want me to commit something ?

**Comment by Julius2342** [comment]
Yay! A pull request is highly appreciated!

**Comment by pulse-mind** [comment]
I did a pull request. 

Also I have a question. 
I'd like to monitor the bus and store some (for some address group) of the values in database. 
I started with the example_telegram_monitor.py. I've seen the method telegram_received_cb where I get a telegram. A telegram has a payload. In some case the payload is a DPTArray. 
1/ how do I get the real value ? like the temperature of the sensor ? 
2/ is that possible to use the devices define in xknx.yaml, watch for telegram or something else (I've seen the def register_device_updated_cb) and then get the value of the device ? 

Thank you
Fred
